gnomad_hail/utils/sample_qc.py

```diff
@@ -278,86 +278,94 @@ def assign_platform_from_pcs(
     return ht
 
 
-# TODO: This should be reviewed / merged with work from Kristen
-def infer_sex(
-        x_mt: hl.MatrixTable,  # TODO: This feels somewhat unsatisfying to provide two MTs. Maybe just reapply the QC MT filters to both (minus callrate for Y)?
-        y_mt: hl.MatrixTable,
-        platform_ht: hl.Table,
-        male_min_f_stat: float,
-        female_max_f_stat: float,
-        min_male_y_sites_called: int = 500,
-        max_y_female_call_rate: float = 0.15,
-        min_y_male_call_rate: float = 0.8
+def default_annotate_sex(
+        mt: hl.MatrixTable,
+        is_sparse: bool = True,
+        excluded_intervals: Optional[hl.Table] = None,
+        included_intervals: Optional[hl.Table] = None,
+        normalization_contig: str = 'chr20',
+        chr_x: Optional[str] = None,
```

Comment:
Because it is a _default_ function, maybe remove `chr_x`, `chr_y` as args? They're super unlikely to be used and also there is a bit of inconsistency in using `x_contigs = get_reference_genome(mt.locus).x_contigs` for the 2nd part of the code if the user provided specific weird contigs..

Reply:
good point! will remove these